helm/README.md

```diff
@@ -0,0 +1,107 @@
+# Agent Check: helm
+
+## Overview
+
+This check monitors [helm][1] through the Datadog Agent.
+
+It mirrors the output values of [helm-exporter][8].  You should expect a metric value of -1
+if the helm chart fails to install.  Here is the complete status map:
+
+```py
+STATUS_MAP = {
+    "unknown": 0,
+    "deployed": 1,
+    "uninstalled": 2,
+    "superseded": 3,
+    "failed": -1,
+    "uninstalling": 5,
+    "pending-install": 6,
+    "pending-upgrade": 7,
+    "pending-rollback": 8,
+}
+```
+
+Because helm uses k8s secrets to store metadata, you'll also need to give the
+pod running the helm check permissions to list secrets.  That will involve
+* A cluster-role that can list secrets
+* A role binding between the cluster role and your pod's service account
+
+It may look like the following:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: a-name-for-your-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: a-name-for-your-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: a-name-for-your-role
+subjects:
+- kind: ServiceAccount
+  name: datadog
+  namespace: datadog
+```
+
+
+## Setup
```

Comment:
Add a note somewhere here about it requiring agent 6.14+ (The first one with the ability to run python 3)